src/test/java/org/myrobotlab/service/ProgramABTest.java

```diff
@@ -308,5 +312,50 @@ public void umlautTest() throws IOException {
     // @GroG says - "this is not working"
     assertEquals("He's a character from Guitar Hero!", resp.msg);
   }
+  
+  @Test
+  public void newTests() throws IOException {
+    // minimal startup - create the service get a response
+    ProgramAB alice = (ProgramAB)Runtime.start("alice", "ProgramAB");
+    assertTrue(alice.getBots().size() > 0);
+    
+    Response response = alice.getResponse("Hello");
```

Comment:
Where is the start session call?  I'd expect that you need to tell it which bot you want to have a conversation with and additionally, which user is talking with the bot.  ProgramAB should not load a default bot unless you start a session with it.

Reply:
start session doesnt even start a bot ..
lazy loading prevents a bot from actually loading unless theres a request for a response